lib/create-webpack-config-base.js

```diff
@@ -61,8 +61,14 @@ function createWebpackConfigBase(batfishConfig) {
         }
       ]
     ].concat(batfishConfig.babelPlugins || []);
-
-    if (!batfishConfig.production) {
+    if (batfishConfig.production) {
+      babelPlugins.push([
+        'transform-react-remove-prop-types',
+        {
+          ignoreFilenames: ['node_modules']
```

Comment:
I don't think we want to include this. mapbox-react-components, for example, will have a lot of prop-types that we'll want to remove. The [reasoning in the docs](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types#is-it-safe) about why you might want to filter out `node_modules` points to what I'd consider a rare and unsafe practice. I think I'd rather apply this to everything, and if we find errors in particular modules ignore those. Does that sound reasonable?